README.md

```diff
@@ -201,3 +201,7 @@ from within the GeoStyler Demo.
 ### <a name="developing-geostyler-data-parsers"></a>Developing GeoStyler Data Parsers
 
 Developing GeoStyler data parsers follows the same pattern as described in [Developing GeoStyler Style Parser](#Developing-GeoStyler-Style-Parsers), but keep in mind that **data parsers have to implement the [GeoStyler Data interface](https://github.com/geostyler/geostyler-data).**
+
+## <a name="license"></a>License
+
+GeoStyler is released under the BSD 2-Clause license. Please see see the file `LICENSE` in the root of this repository.
```

Comment:
You can use a markdown link here if you like:
```
[LICENSE.md](LICENSE.md)
```

Reply:
Oh I see the license is LICENSE, I just assumed LICENCE.md (from https://github.com/IQAndreas/markdown-licenses or similar).

Reply:
Done in 01cb65ccbb0965cf4e3ca5b943be5ba457bd03ce, thanks for the suggestion.